src/main/java/logisticspipes/LogisticsPipes.java

```diff
@@ -494,6 +495,9 @@ private void initItems(Side side) {
         LogisticsPipes.ModuleItem.setUnlocalizedName("itemModule");
         LogisticsPipes.ModuleItem.loadModules();
         GameRegistry.registerItem(LogisticsPipes.ModuleItem, LogisticsPipes.ModuleItem.getUnlocalizedName());
+        if (isClient) {
+            MinecraftForgeClient.registerItemRenderer(LogisticsPipes.ModuleItem, new ItemModuleRenderer());
+        }
```

Comment:
Has this been tested on a server? I suspect that this may cause a server-side crash trying to class-load this. Disregard if this isn't the case as it can be hard to tell sometimes

Reply:
Yes, I have tested it on a LAN server (running on a separate machine). I copypasta'ed this code from existing code (lines 454-456) in the same method, so I assume it is a tested method of registering renderers.